Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -292,11 +297,34 @@ class ButtonClickListener : Object, IOnClickListener
 
 			#endregion
 
-			public void OnClick(global::Android.Views.View v)
+			public void OnClick(AView v)
 			{
 				var renderer = v.Tag as ButtonRenderer;
 				((IButtonController)renderer?.Element)?.SendClicked();
 			}
 		}
+
+		class ButtonTouchListener : Object, IOnTouchListener
+		{
+			#region Statics
+
+			public static readonly Lazy<ButtonTouchListener> Instance = new Lazy<ButtonTouchListener>(() => new ButtonTouchListener());
+
+			#endregion
```

Comment:
I just follow from existing ButtonClickListener implementation. Do you know the reasons why exising ButtonClickListener has region? (line 289 and 293).